cli/cmd/install/install.go

```diff
@@ -162,23 +163,12 @@ func (i *Install) preConfigure(ctx *clicontext.CLIContext, ignoreCluster bool) e
 		return err
 	}
 
-	var totalMemory int64
 	versionWarning := false
 	for _, node := range nodes.Items {
-		totalMemory += node.Status.Capacity.Memory().Value()
 		if !versionWarning {
 			versionWarning = checkKubernetesVersion(node.Status.NodeInfo.KubeletVersion)
 		}
 	}
-	if totalMemory < 2147000000 {
-		if isMinikubeCluster(nodes) {
-			fmt.Println("Warning: detecting that your minikube cluster doesn't have at least 3 GB of memory. Please try to increase memory by running `minikube start --memory 4098`")
```

Comment:
Our install is pretty quiet right now, maybe we should mention we are installing components as they are needed.

Reply:
yep, we should definitely call this out in docs and release notes.